src/pages/Ecommerce/Ecommerce.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Header from '../../components/Header';
+
+function Ecommerce() {
+  return (
+    <div>
+      <Header title="Ecommerce" />
```

Comment:
All Header elements should be outside of the pages, in App.jsx. Fix it for all 5 projects.

Reply:
We discused to drop this change